current/Dockerfile

```diff
@@ -4,8 +4,8 @@
 FROM bitnami/minideb:stretch
 RUN install_packages curl ca-certificates
 
-ENV SONAR_SCALA_VERSION 8.4.0
-ENV GIT_PLUGIN_VERSION 1.11.1.2008
+ENV SONAR_SCALA_VERSION 8.5.0
+ENV GIT_PLUGIN_VERSION 1.12.0.2034
```

Comment:
I need to downgrade this back to `1.11.1.2008` as the plugin manifest seems to be missing in `1.12.0.2034`:
> java.lang.IllegalStateException: Unable to read plugin manifest from jar : /opt/sonarqube/extensions/plugins/sonar-scm-git-plugin-1.12.0.2034.jar

Reply:
Actually, it appears that 1.12 was not pushed to maven yet.